website/pages/docs/concepts/seal.mdx

```diff
@@ -89,75 +89,159 @@ Recovery Keys and are used for these operations.
-keys and Auto Unseal to another Auto Unseal.
+The seal can be migrated from Shamir Seal to KMS Seal, KMS Seal to Shamir Seal
+and KMS Seal to another KMS Seal.
+
+~> **NOTE**: The seal migration process cannot be performed without downtime.
+Due to the intricate technical underpinnings of the seal feature, it is at this
+point not possible to perform seal migration without bringing the whole cluster
+down. We understand that it can be painful for many users to bring the full
+cluster down. But, we believe that switching seals is a very rare event and
+hence hope for the downtime to be considered as an acceptable trade-off.
 
 ~> **NOTE**: The migration operation will require both seals to be available
-during the migration. For example, a migration from a cloud KMS seal to
-Shamir will require that the cloud KMS be accessible during the migration.
+during the migration. For example, a migration from a KMS seal to Shamir will
+require that the KMS key be accessible during the migration.
+
+### Migration From Shamir Seal to KMS Seal
+
+#### When **not** using Integrated Storage
 
-### Migration From Shamir to Auto Unseal
+1. Take your server cluster offline.
 
-To migrate from Shamir keys to Auto Unseal, take your server cluster offline and update
-the [seal configuration](/docs/configuration/seal) with the appropriate seal
-configuration. Bring your server back up and leave the rest of the nodes offline if
-using multi-server mode, then run the unseal process with the `-migrate` flag and bring
-the rest of the cluster online.
+2. Add the desired KMS [seal block](/docs/configuration/seal) to the
+configuration file of one of the nodes.
 
-All unseal commands must specify the `-migrate` flag. Once the required threshold of
-unseal keys are entered, unseal keys will be migrated to recovery keys.
+3. Bring this one node back up and run the unseal command by supplying the
+`-migrate` flag for each unseal key. This node will be the active node and
+completes the seal migration process. The Shamir unseal keys supplied here will
+be migrated as the recovery keys for the KMS seal.
 
-```
-$ vault operator unseal -migrate
-```
+4. Standby nodes can join the active node's cluster using the new seal blocks.
+Note that by the time standby nodes join the cluster, the migration process is
+already complete. Hence, the standby nodes will be auto-unsealed through the KMS
+key and will not require an explicit unseal command.
 
-### Migration From Auto Unseal to Shamir
+#### When using Integrated Storage
+
+Integrated Storage is implemented using the [Raft Consensus Algorithm][raft] and
+would require a quorum of nodes to be fully operational.
+
+1. Take your server cluster offline.
+
+2. Add the desired KMS [seal block](/docs/configuration/seal) to the
+configuration files of quorum number of nodes in the cluster.
+
+3. Bring the quorum number of nodes back up. On each of these nodes, run the
```

Comment:
Maybe break this into two steps - the first bringing the nodes back up, and the second running the unseal command on each of the nodes (unless the order is bring them back up and run the unseal command on each before bringing the next up, in which case, I think we need to be more explicit).